packages/plugin-kafka/README.md

```diff
@@ -64,12 +69,12 @@ consumerStream = await service.kafka.createConsumerStream({
   streamOptions: { topics: topic, streamAsBatch: true, fetchSize: 10 },
   conf: {
     debug: 'consumer',
-    'auto.commit.enable': false,
+    'enable.auto.commit': false,
     'client.id': 'someid',
     'group.id': 'other-group',
   },
   topic: {
-    'auto.offset.reset': 'earliest',
+    'auto.offset.reset': 'largest', // in case with earliest it ignores commited vfalues
```

Comment:
actually still needs earlier, as it will take the last message from the topic otherwise